deployment/scripts/run_dcl_node

```diff
@@ -220,12 +222,12 @@ function config_cli {
     local _chain_id="$1"
     local _peer_host="${2:-}"
 
-    dcld config chain-id "$_chain_id"
-    dcld config output json
-    dcld config broadcast-mode block
+    ./dcld config chain-id "$_chain_id"
```

Comment:
Before cosmovisor integration, `dcld` was copied to `/usr/bin` and so it became available for execution as a command from any directory.

Now `cosmovisor` instead of `dcld` is copied to `/usr/bin`, while `dcld` is copied to a cosmovisor-specific directory (`$HOME/.dcl/cosmovisor/genesis/bin`). So now `dcld` does not become available as a command at this step. So we have to keep it in the current directory too for performing initialization steps.

Further, after cosmovisor has been started and it has created `current` symbolic link, the command adding `$HOME/.dcl/cosmovisor/current/bin` to PATH is added to `$HOME/.profile` and the latter is re-executed, so `dcld` becomes available as a command from any directory after that step.

Added a clarification to _Setup DCL binaries_ section in `docs/running-node.md` that `dcld` must not be removed from the current working directory.